build_tools/integrate/README.md

```diff
@@ -0,0 +1,85 @@
+# Integrate Scripts
+
+A collection of scripts used to integrate the StableHLO repository into the
+rest of the OpenXLA repos. These scripts are run ~2x/wk by a StableHLO oncall
+rotation at Google to ensure new changes can propagate to the rest of the
+ecosystem in a reasonable amount of time.
+
+## Integrate Process
+
+### Bump LLVM Revision
+
+First we bump LLVM to match that of XLA, and apply any patches the the XLA
+LLVM integrate had to apply to StableHLO to build LLVM:
```

Comment:
nit: StableHLO at google monorepo ?

Reply:
Given that this bit can be run externally, going to keep the mono-repo out of this. That said I've reworked the wording to make it more clear how this all fits together with the XLA repo